users/tests/test_user.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+from django.contrib.auth.models import User
+from django.urls import reverse
+import uuid
+
+
+@pytest.fixture
+def test_password():
+    return 'strong-test-pass'
+
+
+@pytest.fixture
+def test_username():
+    return 'Test-user'
+
+
+@pytest.fixture
+def test_email():
+    return 'Test-user@mail.com'
+
+
+@pytest.fixture
+def user(test_username, test_email, test_password):
+    return User.objects.create_user(test_username, test_email, test_password)
+
+
+@pytest.fixture
+def create_user(db, django_user_model, test_password):
+    def make_user(**kwargs):
+        kwargs['password'] = test_password
+        if 'username' not in kwargs:
+            kwargs['username'] = str(uuid.uuid4())
+        return django_user_model.objects.create_user(**kwargs)
+
+    return make_user
+
+
+@pytest.mark.django_db
+def test_user_creation(user):
+    assert User.objects.filter(pk=user.id).exists()
+    assert User.objects.get(pk=user.id) == user
+
+
+@pytest.mark.django_db
+def test_user_delete(user):
+    assert User.objects.filter(pk=user.id).exists()
+    User.objects.get(pk=user.id).delete()
+    assert not User.objects.filter(pk=user.id).exists()
+
+
+@pytest.mark.django_db
+def test_should_not_check_unusable_password(user):
+    user.set_unusable_password()
+    assert not user.has_usable_password()
+
+
+def test_with_authenticated_client(client, django_user_model):
+    username = "admin"
+    password = "123456"
+    django_user_model.objects.create_user(username=username, password=password)
+    client.login(username=username, password=password)
+    response = client.get(reverse('profile'))
```

Comment:
please use w/out reverse /profile similar to App/uri.py

Reply:
I made the change